tests/foundry/foundry.k.check.expected

```diff
@@ -3008,64 +3008,157 @@ module LOOPSTEST-BIN-RUNTIME
+    
+    syntax MethodDisambiguateTestField ::= "_failed" [symbol(), klabel(field_MethodDisambiguateTest__failed)]
+    
+    syntax MethodDisambiguateTestField ::= "IS_SCRIPT" [symbol(), klabel(field_MethodDisambiguateTest_IS_SCRIPT)]
+    
+    syntax MethodDisambiguateTestField ::= "stdstore" [symbol(), klabel(field_MethodDisambiguateTest_stdstore)]
+    
+    rule  ( #loc ( MethodDisambiguateTest . IS_TEST ) => 0 )
+      
+    
+    rule  ( #loc ( MethodDisambiguateTest . _failed ) => 0 )
+      
+    
+    rule  ( #loc ( MethodDisambiguateTest . IS_SCRIPT ) => 0 )
+      
+    
+    rule  ( #loc ( MethodDisambiguateTest . stdstore ) => 1 )
+      
+    
+    syntax Bytes ::= MethodDisambiguateTestContract "." MethodDisambiguateTestMethod [function(), symbol(), klabel(method_MethodDisambiguateTest)]
+    
+    syntax MethodDisambiguateTestMethod ::= "IS_SCRIPT" "(" ")" [symbol(), klabel(method_MethodDisambiguateTest_IS_SCRIPT)]
+    
+    syntax MethodDisambiguateTestMethod ::= "IS_TEST" "(" ")" [symbol(), klabel(method_MethodDisambiguateTest_IS_TEST)]
+    
+    syntax MethodDisambiguateTestMethod ::= "failed" "(" ")" [symbol(), klabel(method_MethodDisambiguateTest_failed)]
+    
+    syntax MethodDisambiguateTestMethod ::= "getNumber" "(" Int ")" [symbol(), klabel(method_MethodDisambiguateTest_getNumber)]
+    
+    syntax MethodDisambiguateTestMethod ::= "getNumber" "(" Int ")" [symbol(), klabel(method_MethodDisambiguateTest_getNumber)]
+    
+    syntax MethodDisambiguateTestMethod ::= "test_method_call" "(" ")" [symbol(), klabel(method_MethodDisambiguateTest_test_method_call)]
+    
+    syntax MethodDisambiguateTestMethod ::= "vm" "(" ")" [symbol(), klabel(method_MethodDisambiguateTest_vm)]
+    
+    rule  ( MethodDisambiguateTest . IS_SCRIPT ( ) => #abiCallData ( "IS_SCRIPT" , .TypedArgs ) )
+      
+    
+    rule  ( MethodDisambiguateTest . IS_TEST ( ) => #abiCallData ( "IS_TEST" , .TypedArgs ) )
+      
+    
+    rule  ( MethodDisambiguateTest . failed ( ) => #abiCallData ( "failed" , .TypedArgs ) )
+      
+    
+    rule  ( MethodDisambiguateTest . getNumber ( V0_x ) => #abiCallData ( "getNumber" , #uint32 ( V0_x ) , .TypedArgs ) )
+       ensures #rangeUInt ( 32 , V0_x )
+      
+    
+    rule  ( MethodDisambiguateTest . getNumber ( V0_x ) => #abiCallData ( "getNumber" , #uint32 ( V0_x ) , .TypedArgs ) )
+       ensures #rangeUInt ( 32 , V0_x )
+      
+    
+    rule  ( MethodDisambiguateTest . test_method_call ( ) => #abiCallData ( "test_method_call" , .TypedArgs ) )
+      
+    
+    rule  ( MethodDisambiguateTest . vm ( ) => #abiCallData ( "vm" , .TypedArgs ) )
+      
+    
+    rule  ( selector ( "IS_SCRIPT()" ) => 4174167879 )
+      
+    
+    rule  ( selector ( "IS_TEST()" ) => 4202047188 )
+      
+    
+    rule  ( selector ( "failed()" ) => 3124842406 )
+      
+    
+    rule  ( selector ( "getNumber(uint32)" ) => 4233508440 )
+      
+    
+    rule  ( selector ( "getNumber(uint32)" ) => 144150998 )
```

Comment:
Hmmmm, are we extracting the signature correctly? One of these should have `uint256`.

Reply:
It turns out it was being caused by another spot in the code that assumed methods have unique names. It should be fixed now.